src/gantry/screens.py

```
@@ -1110,469 +1111,46 @@ def on_search_input_search_changed(self, message: SearchInput.SearchChanged) ->
-    def _display_detail_panel(self, content: str) -> None:
-        """Display content in the detail panel."""
+    def on_directory_tree_file_selected(
+        self, event: DirectoryTree.FileSelected
+    ) -> None:
+        path = event.path
         try:
-            detail_panel = self.query_one("#detail-panel", Label)
-            detail_panel.update(content)
-            detail_panel.add_class("show")
+            content = path.read_text(errors="replace")
         except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)